LICENSE_FAQ.md

```diff
@@ -0,0 +1,151 @@
+
+# License FAQ
```

Comment:
fix markdown linter

Reply:
License issues addressed on [this PR](https://github.com/evmos/evmos/pull/1537)